src/CasingOfString.md

```diff
@@ -92,3 +93,33 @@ func CheckCase(input string) string {
 	}
 }
 ```
+
+### Python
+```python
+import unicodedata
```

Comment:
See if you can do it without using any lib so the function can be used by anyone

Reply:
Fixed